### DISCUSSION TITLE
Multisheet  chart problem

### DISCUSSION THREAD

**Discussion by janpernik** [jxlsteam/jxls#135] [Q&A] [answered]
Hi, we are using multisheet. But charts are not copied from template to final sheet?
How to solve it?
Thnx
Jan

**Comment by SoltauFintel** [comment, marked as answer]
I guess(!) the combination Multisheet+Charts is not supported.

**Reply by leonate** [reply]
POI still has rather limited support for charts http://poi.apache.org/components/spreadsheet/limitations.html, so I doubt it is possible to copy charts from one sheet to another with the POI API.
